lib/solidus_support/engine_extensions.rb

```diff
@@ -110,6 +110,7 @@ def enable_solidus_engine_support(engine)
           path = root.join("lib/decorators/#{engine}")
 
           config.autoload_paths += path.glob('*')
+          config.autoload_paths += root.join("lib/controllers/#{engine}").glob('*')
```

Comment:
Can I suggest this version to be as clear as possible about what we are doing? This part is already a puzzle and I think a little bit of extra clarity (for free) will help future readers.

```suggestion
          decorators_path = root.join("lib/decorators/#{engine}")
          controllers_path = root.join("lib/controllers/#{engine}")

          config.autoload_paths += decorators_path.glob('*')
          config.autoload_paths += controllers_path.glob('*')
          
          engine_context = self
          config.to_prepare do
            engine_context.instance_eval do
              load_solidus_decorators_from(decorators_path)
            end
          end
```

Reply:
@kennyadsl I have applied your changes